upgrade/UPGRADE-v8.1.0-dev.md

```diff
@@ -203,7 +203,7 @@ There you can find links to upgrade notes for other versions too.
                     script:
                         - sudo apt install ruby
                         - gem install docker-sync
-                        - sed -i -r "s#sed -i -E#sed -i -r#" ./scripts/install.sh
+                        - sed -i -r "s#sed -i \'\' -E#sed -i -r#" ./scripts/install.sh
```

Comment:
Is it needed when it's running on Linux? And shouldn't it be without backslash as it's in .travis file?

Reply:
It is needed.
You are right about the escaping! Thanks.